projects/kit/components/routable-dialog/generate-lazy-dialogable-route.ts

```diff
@@ -0,0 +1,19 @@
+import {Type} from '@angular/core';
+import {Route} from '@angular/router';
+import {TuiDialogOptions} from '@taiga-ui/core';
+
+import {TuiRoutableDialogComponent} from './routable-dialog.component';
+
+export function tuiGenerateLazyDialogableRoute<I>(
```

Comment:
Do we really need two methods? Seems like they only differ by `path`. Maybe just make `path` second argument, defaulting to empty line? Or make `options: Partial<TuiDialogOptions<I>> & { path?: string }`.

Reply:
Sounds good. Will fix

Reply:
Just gotta make sure `backUrl` is properly calculated for all cases.

Reply:
Upgraded `tuiGenerateDialogableRoute` for supporting lazy loading case. Add tests for all cases